webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -25,6 +37,7 @@ import RepoIcon from './RepoIcon';
 import SelectableDropdown from '../../../../components/SelectableDropdown';
 import { capitalize } from '../../../../utils/helpers';
 
+
```

Comment:
I thought eslint picked these things up, but I don't think this blank line is needed